src/agoradatatools/etl/transform/gene_info.py

```diff
@@ -61,33 +61,42 @@ def transform_gene_info(
     druggability = druggability[useful_columns]
 
     target_list = nest_fields(
```

Comment:
Note: changes like this are from the `black` formatter, I didn't change anything functional.